test/jdk/java/util/stream/boottest/java.base/java/util/stream/SpinedBufferTest1.java

```diff
@@ -26,15 +26,15 @@
 import org.testng.annotations.Test;
 
 import java.util.*;
-import java.util.function.DoubleConsumer;
+//import java.util.function.DoubleConsumer;
```

Comment:
These commented out imports should go away.

Reply:
Sorry about that. I didn't mean to leave cruft behind.